tools/mlaunch/Microsoft.DotNet.Mlaunch.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<!-- This file is used to pack the already-built mlaunch as a .nupkg -->
+
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+		<!-- Suppresses the warnings about the package not having assemblies in lib/*/.dll.-->
+		<NoPackageAnalysis>true</NoPackageAnalysis>
+		<VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
I think we should come up with a better version than 1.0.0.

We could use the existing mlaunch version:

```shell
$ /Library/Frameworks/Xamarin.iOS.framework/Versions/Current/bin/mlaunch --version 
Xamarin.Launcher 14.21.0.330 (xamarin/main: cddbd1915d)
```

or we could come up with something else (Xcode version? Something completely different that's calculated in maccore and doesn't take xamarin-macios into account? Go with "0.0.1" and decide something later?)

@spouliot / @dalexsoto what do you think? I lean towards the mlaunch version (although the downside here is that there can be multiple mlaunch versions for the same maccore hash).

Reply:
IMO it would be useful to use the Xcode version for which `mlaunch` was built (and known to work [1]) for. We should also print that version (at least for the nuget version, maybe not the one we ship bundled in the legacy SDK)

[1] and _should_ work on earlier Xcode too...

Using legacy version is confusing as 14.21 means almost nothing, except iOS 14.x -> Xcode 12.x
That already requires mapping (-2) but only works for the major version.

Using 1.0.0 would require a new totally mapping, ending up in more confusion.

Reply:
I still see 1.0.0 here ?

Reply:
@spouliot there's a default that needs to be in (for example if you build it locally). We override that in the CI:
```bash
"$DOTNET6" pack --version-suffix "$MACCORE_HASH" /p:VersionPrefix=$XCODE_VERSION
```